### PR TITLE
Release notes for 2.29.0.

### PR DESCRIPTION
**Pull request by rstam**

### CODE REVIEW

File: Release Notes/Release Notes v2.29.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# .NET Driver Version 2.29.0 Release Notes
+
+This is the general availability release for the 2.29.0 version of the driver.
+
+Version 2.29.0 of the driver has been tested against MongoDB Server version 8.0 and adds support for the following new features in server version 8.0:
+
++ Support for v2 Queryable Encryption range protocol - [CSHARP-4959](https://jira.mongodb.org/browse/CSHARP-4959)
++ Support Range Indexes - [CSHARP-5057](https://jira.mongodb.org/browse/CSHARP-5057)
```
Comment (BorisDog) [inline Release Notes/Release Notes v2.29.0.md:8]: @rishitb-mongodb  "Support Range Indexes (as?) GA"?

Comment (rishitb-mongodb) [reply Release Notes/Release Notes v2.29.0.md:8]: Yup, I think we can say 
- Support for v2 of the Queryable Encryption range protocol. 
- Range indexes for Queryable Encryption are now GA

**Review by BorisDog — APPROVED** [review]
LGTM

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request closed without merging**